client/landing/gutenboarding/onboarding-block/fse-beta-opt-in/index.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+	Title,
+	SubTitle,
+	ActionButtons,
+	NextButton,
+	BackButton,
+	SkipButton,
+} from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { Icon, siteLogo, header, navigation, arrowRight } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import useStepNavigation from '../../hooks/use-step-navigation';
+import { useTrackStep } from '../../hooks/use-track-step';
+import { trackEventWithFlow } from '../../lib/analytics';
+import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
+
+import './style.scss';
+
+const FseBetaOptIn: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const { goNext, goBack } = useStepNavigation();
+	const { enrollInFseBeta } = useDispatch( ONBOARD_STORE );
+	const { shouldEnrollInFseBeta } = useSelect( ( select ) => select( ONBOARD_STORE ) );
+	const pickBeta = ( shouldEnroll: boolean ) => {
+		enrollInFseBeta( shouldEnroll );
+		goNext();
+	};
+	const optIn = () => {
+		trackEventWithFlow( 'calypso_fse_beta_opt_in' );
+		pickBeta( true );
+	};
+
+	useTrackStep( 'FseBetaOptIn', () => ( {
+		selected_fse_beta_opt_in: shouldEnrollInFseBeta(),
+	} ) );
+
+	return (
+		<div className="gutenboarding-page fse-beta-opt-in">
+			<div className="fse-beta-opt-in__header">
+				<div>
+					<Title>{ __( 'Join Our Full Site Editing Beta' ) }</Title>
+					<SubTitle>{ __( 'Experience the new WordPress.com site editing features' ) }</SubTitle>
+				</div>
+				<ActionButtons className="fse-beta-opt-in__actions">
+					<BackButton onClick={ goBack } />
+					<SkipButton onClick={ () => pickBeta( false ) } />
```

Comment:
Just a random thought: do you think it would make sense to also track people who skip (opt-out) the beta step? 🤔

Reply:
Yeah I went back and forth and it looked like `useTrackStep` would track both so I didn't bother but who am I kidding I have no idea how there work, let's track both to be sure